src/pages/responsible-trading/_need-help.js

```diff
@@ -20,7 +22,8 @@ const help_content = [
         text2: (
             <Localize translate_text="GAMSTOP is a free service that enables you to self-exclude from all online gambling companies licensed in Great Britain." />
         ),
-        icon: <img src={GamStop} alt="gamstop" style={{ width: '91%', height: '100%' }} />,
+        icon: <img src={GamStop} alt="gamstop" width="160px" height="64px" />,
+        icon_mobile: <img src={GamStop} alt="gamstop" width="113px" height="34px" />,
```

Comment:
```suggestion
        icon: <img src={GamStop} alt="gamstop" width="160" height="64" />,
        icon_mobile: <img src={GamStop} alt="gamstop" width="113" height="34" />,
```

`width` and `height` shouldn't have trailing `px` please refer to [this](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/) article